Makefile

```diff
@@ -4,7 +4,7 @@ DEVELOPMENT?=false
 AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 AWS_REGION?=us-west-2
 IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
-BASE_IMAGE?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com/eks-distro/base:2021-01-05-1609822390
+BASE_IMAGE?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com/eks-distro/base:$(shell cat EKS_DISTRO_BASE_TAG_FILE)
```

Comment:
Let's separate out the variable and have that overridable instead, like BASE_IMAGE_TAG or BASE_TAG like below

Reply:
Sure